src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig

```diff
@@ -1 +1,24 @@
-Welcome!
+<div class="card">
+    <div class="card-body">
+        <div class="jumbotron p-1 bg-white mb-0">
+            <h1 class="display-4">Welcome</h1>
+            <p class="lead">in your Chameleon System open source backend</p>
+            <hr class="my-4">
+            <p>
+                Visit our product website for information about Chameleon System and available extensions.
```

Comment:
```suggestion
                Visit our product website for information on Chameleon System and available extensions.
```

Reply:
done

Reply:
You think you are smarter than deepl.com, eh? :-)

Reply:
StackExchange has our back: https://english.stackexchange.com/questions/32838/what-is-the-difference-between-information-on-information-of-information

So decide yourself if the information provided on our websites is more in-depth or more brochure-like :-)

Reply:
As this is the last comment left I'd say I merge this PR. If you like "about" better than "on", just file a new mini-PR :-)